src/main/resources/openapi/schemas/tokenResponse.json

```diff
@@ -0,0 +1,19 @@
+{
+  "description": "The signed JWT token based on the payload",
```

Comment:
description doesn't match properties

Reply:
Agreed. Changed to better describe how the schema is used. Its either a `token` (when it's a dummy signing request) or both a `refreshToken` and an `accessToken` when it is a "regular" signing request.